docs/content/configuration/configuring_hostdefiner.md

```diff
@@ -0,0 +1,12 @@
+# Configuring the host definer
+
+Some of the parameters within the HostDefiner custom resource are configurable. Use this information to help decide whether the parameters for your storage system need to be updated.
+
+For more information about using the host definer, see [Using dynamic host definition](../using/using_hostdefinition.md).
+    
+|Field|Description|
+|---------|--------|
+|`prefix`|Adds a prefix to the hosts defined by the host definer.|
+|`connectivity`|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvme` for use with NVME over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br><br>By default, this field is blank and the host definer selects the first of available connectivity types on the node, according to the following hierarchy: NVMe, FC, iSCSI.|
+|`allowDelete`|Defines whether the host definer is allowed to delete host definitions.<br>Input values are `true` or `false`.<br>The default value is `true`.|
+|`dynamicNodeLabeling`|Defines whether the nodes that run the CSI node pod are dynamically labeled or if the user must create the `hostdefiner.block.csi.ibm.com/managed-by=true` label on each relevant node. This label tells the host definer which nodes to manage their host definition on the storage side.<br>Input values are `true` or `false`.<br>The default value is `false`, where the user must create each label.|
```

Comment:
Also need to add note that these labels won't be deleted when hostdefiner will be removed, and explain on which situation the label will be deleted automatically when the value set to "true".
Also there should be a note on a case which the label/s won't succeed to be deleted see - https://jira.xiv.ibm.com/browse/CSI-4446

Reply:
is CSI-4446 a known issue or just part of the feature currently?

Reply:
@roysahar I asked @erantzabari about what to write about the labels won't be deleted when hostdefiner will be removed, and he said that we need to add this note at the uninstall process and it's enough
![image](https://user-images.githubusercontent.com/45543087/178969728-f24fe0ab-393d-416d-a883-85f0d9eb2cc8.png)

Reply:
"and explain on which situation the label will be deleted automatically when the value set to "true"."


what is the specific situation that i should say ?

Reply:
Currently I already updated 
![image](https://user-images.githubusercontent.com/54570654/178971513-e7bbb029-f155-409c-aa72-954927278d51.png)

should this be removed? should I do any other updates for this section or per Matan this is ok as is?

@matancarmeli7  @roysahar  thanks

Reply:
in my meeting with @roysahar  and @matancarmeli7  We removed the note in the end that was added to the parameter.
Needs to be decided by @erantzabari : should https://jira.xiv.ibm.com/browse/CSI-4446 be listed as a known issue / limitation in the guide.

Reply:
@rivkap please avoid using internal links in public. a ticket number should be enough

Reply:
@roysahar  can we close this thread?

Reply:
done.